[2024-05-07 20:11:17,973] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: research_scraper.research_scraper manual__2024-05-07T20:03:45.915344+00:00 [queued]>
[2024-05-07 20:11:17,983] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: research_scraper.research_scraper manual__2024-05-07T20:03:45.915344+00:00 [queued]>
[2024-05-07 20:11:17,983] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-05-07 20:11:17,983] {taskinstance.py:1357} INFO - Starting attempt 2 of 2
[2024-05-07 20:11:17,984] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-05-07 20:11:18,001] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): research_scraper> on 2024-05-07 20:03:45.915344+00:00
[2024-05-07 20:11:18,012] {standard_task_runner.py:52} INFO - Started process 1384 to run task
[2024-05-07 20:11:18,019] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'research_scraper', 'research_scraper', 'manual__2024-05-07T20:03:45.915344+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/ResearchScraper.py', '--cfg-path', '/tmp/tmpwnokpfp1', '--error-file', '/tmp/tmpy0ds812y']
[2024-05-07 20:11:18,020] {standard_task_runner.py:80} INFO - Job 203: Subtask research_scraper
[2024-05-07 20:11:18,096] {task_command.py:369} INFO - Running <TaskInstance: research_scraper.research_scraper manual__2024-05-07T20:03:45.915344+00:00 [running]> on host 5881f5798927
[2024-05-07 20:11:18,179] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=research_scraper
AIRFLOW_CTX_TASK_ID=research_scraper
AIRFLOW_CTX_EXECUTION_DATE=2024-05-07T20:03:45.915344+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-07T20:03:45.915344+00:00
[2024-05-07 20:11:18,182] {conn.py:381} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('192.168.228.5', 9092)]>: connecting to kafka:9092 [('192.168.228.5', 9092) IPv4]
[2024-05-07 20:11:18,183] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2024-05-07 20:11:18,183] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:9092 <connecting> [IPv4 ('192.168.228.5', 9092)]>: Connection complete.
[2024-05-07 20:11:18,293] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2024-05-07 20:11:18,294] {conn.py:1269} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2024-05-07 20:11:18,299] {logging_mixin.py:115} INFO - ['2']
[2024-05-07 20:11:18,299] {taskinstance.py:1889} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/ResearchScraper.py", line 30, in call_scopus
    get_scopus_data()
  File "/opt/airflow/dags/scopus.py", line 64, in get_scopus_data
    start_pointer = read_start_pointer()
  File "/opt/airflow/dags/scopus.py", line 36, in read_start_pointer
    result[f'{row[0].strip()}{row[1].strip()}']= int(row[2].strip())
IndexError: list index out of range
[2024-05-07 20:11:18,309] {taskinstance.py:1400} INFO - Marking task as FAILED. dag_id=research_scraper, task_id=research_scraper, execution_date=20240507T200345, start_date=20240507T201117, end_date=20240507T201118
[2024-05-07 20:11:18,319] {standard_task_runner.py:97} ERROR - Failed to execute job 203 for task research_scraper (list index out of range; 1384)
[2024-05-07 20:11:18,327] {local_task_job.py:156} INFO - Task exited with return code 1
[2024-05-07 20:11:18,373] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
